src/utils/cryptography.js

```diff
@@ -15,97 +15,60 @@
  */
 import elements from 'lisk-elements';
 
-const wrapFunction = fn =>
-	function wrappedFunction(...args) {
-		try {
-			return fn(...args);
-		} catch ({ message: error }) {
-			return { error };
-		}
-	};
+const liskCrypto = elements.cryptography;
 
-class Crypto {
-	constructor() {
-		this.liskCrypto = elements.cryptography;
+export const encryptMessage = ({ message, passphrase, recipient }) =>
+	liskCrypto.encryptMessageWithPassphrase(message, passphrase, recipient);
 
-		[
-			'encryptMessage',
-			'decryptMessage',
-			'encryptPassphrase',
-			'decryptPassphrase',
-			'getKeys',
-			'getAddressFromPublicKey',
-			'signMessage',
-			'verifyMessage',
-		].forEach(methodName => {
-			this[methodName] = wrapFunction(this[methodName].bind(this));
-		});
-	}
+export const decryptMessage = ({
+	cipher,
+	nonce,
+	passphrase,
+	senderPublicKey,
+}) => ({
+	message: liskCrypto.decryptMessageWithPassphrase(
+		cipher,
+		nonce,
+		passphrase,
+		senderPublicKey,
+	),
+});
 
-	encryptMessage({ message, passphrase, recipient }) {
-		return this.liskCrypto.encryptMessageWithPassphrase(
-			message,
-			passphrase,
-			recipient,
-		);
-	}
+export const encryptPassphrase = ({ passphrase, password }) => {
+	const encryptedPassphraseObject = liskCrypto.encryptPassphraseWithPassword(
+		passphrase,
+		password,
+	);
+	const encryptedPassphrase = liskCrypto.stringifyEncryptedPassphrase(
+		encryptedPassphraseObject,
+	);
+	return { encryptedPassphrase };
+};
 
-	decryptMessage({ cipher, nonce, passphrase, senderPublicKey }) {
-		return {
-			message: this.liskCrypto.decryptMessageWithPassphrase(
-				cipher,
-				nonce,
-				passphrase,
-				senderPublicKey,
-			),
-		};
-	}
+export const decryptPassphrase = ({ encryptedPassphrase, password }) => {
+	const encryptedPassphraseObject = liskCrypto.parseEncryptedPassphrase(
+		encryptedPassphrase,
+	);
+	const passphrase = liskCrypto.decryptPassphraseWithPassword(
+		encryptedPassphraseObject,
+		password,
+	);
+	return { passphrase };
+};
 
-	encryptPassphrase({ passphrase, password }) {
-		const encryptedPassphraseObject = this.liskCrypto.encryptPassphraseWithPassword(
-			passphrase,
-			password,
-		);
-		const encryptedPassphrase = this.liskCrypto.stringifyEncryptedPassphrase(
-			encryptedPassphraseObject,
-		);
-		return { encryptedPassphrase };
-	}
+export const getKeys = passphrase => liskCrypto.getKeys(passphrase);
```

Comment:
Do we need there wrap functions? it's like re-exporting the elements feature.
Since it's `external` library, maybe... but im not so sure

Reply:
I'm skipping this as we discussed to remove these in future.